physioqc/metrics/utils.py

```diff
@@ -85,3 +91,349 @@ def has_peaks(signal: Physio) -> bool:
         LGR.warn("Signal has only one peak! Better to rerun peak detection.")
 
     return signal._metadata["peaks"].size > 1
+
+
+# - butterworth filters
+# @conditionaljit()
+def butterlpfiltfilt(Fs, upperpass, inputdata, order, debug=False):
```

Comment:
Since peakdet is a dependency of physioqc, I'm wondering if it wouldn't be easier to just use the [peakdet.operations.filter_physio](https://peakdet.readthedocs.io/en/latest/api.html#peakdet.operations.filter_physio) function, which already supports low/high/band-pass filtering

Reply:
I've converted chest_belt.py to use peakdet routines for filtering and interpolation.